src/enums/filing-codes.ts

```diff
@@ -1,12 +1,13 @@
-/** Filing codes to send to pay. */
+/** Filing codes to send to Pay API. */
 export enum FilingCodes {
   ADDRESS_CHANGE_BC = 'BCADD', // BCOMP - Change of Address
   ADDRESS_CHANGE_OT = 'OTADD', // Others - Change of Address
   AGM_EXTENSION = 'AGMDT',
   AGM_LOCATION_CHANGE = 'AGMLC',
   AFFIDAVIT = 'AFDVT',
   ALTERATION = 'ALTER',
-  ALTERATION_BC_TO_ULC = 'NOALU', // More expensive for BC to ULC.
+  ALTERATION_BC_TO_ULC = 'NOALU', // different fee for BC to ULC
+  AMALGAMATION = 'AMALG',
```

Comment:
I'm still waiting for confirmation on this (or multiple) amalgamation fee code(s).

Reply:
We might have a different fee code depending on the amalgamation type (short, regular etc...)?

Reply:
We now have 3 different filing codes (reg, horiz, vert). Resolving this conversation.